### DISCUSSION TITLE
vrcx  SyntaxError: Unexpected end of JSON input help ? :)

### DISCUSSION THREAD

**Discussion by Vandal888** [vrcx-team/VRCX#524] [Q&A] [answered]
I can't search avatars for 3 weeks now ...and when i try to search for any Avatars i always get the message 

https://api.ares-mod.com/vrcx_search.php SyntaxError: Unexpected end of JSON input

does anyone have a solution for this ? :)

**Comment by Myrkie** [comment, marked as answer]
Dont use ares search, the only supported search is at https://avtr.just-h.party/

**Reply by Vandal888** [reply]
oooh ok.... and are there any other Avatar Database Provider than https://avtr.just-h.party/ or is this the only one working right now ?

**Reply by Myrkie** [reply]
all database providers would be kind of pointless considering the avtr.just-h.party is culmination of multiple older avatar databases that have shut down and have been combined together, it is also self updating and ever expanding. that is the only one that vrcx officially supports and works with is creators to maintain support, any others such as ares-mod are not working with the VRCX Team and if they break is out of the control of VRCX team

**Reply by Vandal888** [reply]
ok thanks for the quick reply :)
